_proposals/pup-0001.rst

```diff
@@ -0,0 +1,207 @@
+---
+title: PUP Purpose and Guidelines
+author: Brian Bouterse
+---
+
+Summary
+=======
+
+This proposal is designed to
+
+Motivation
+==========
+
+As initially [raised on the pulp-dev list](https://www.redhat.com/archives/pulp-dev/2017-February/msg00014.html),
+Pulp lacks a process to propose, track, and decide on changes that affect the Pulp community.
+
+This is a problem that has been addressed by many communities. The process outlined in this document
+is inspired by processes in the Foreman, Python, and Django communities.
+
+
+What are these called?
+======================
+
+We need an acronym or initialization for these. Here are some quick ideas:
+
+* "Pulp Update Proposal" (PUP)
+* "Pulp Change Proposal" (PCP)
+* <Your idea here>
+
+
+Tracking Proposals
+==================
+
+Proposals would be in restructured text format and sent as a pull request against the
+pulpproject.org repo or a dedicated repo. We could easily create a tab at the top of pulpproject.org
+to show off merged proposals. This could also be in a separate, dedicated repo like Python does, but
+for simplicity I'm modeling this using the pulpproject.org repo. The idea to render proposals is
+similar to how [Python Enhancement Proposals (PEPs)](https://www.python.org/dev/peps/pep-0001/)
+work, which are markdown documents that are also viewable/readable as rendered documents via a
+browser.
+
+
+Finding/Discovering Proposals
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An index mimicking [the Python one](https://www.python.org/dev/peps/) would be created which would
+show all proposals in different states (active, approved, rejected, abandoned). These would also be
+viewable via pulpproject.org in a dedicated area.
+
+
+Proposal Lifecycle
+==================
+
+Before Proposing
+^^^^^^^^^^^^^^^^
+
+Discussing an idea via the list prior to writing a formal proposal is encouraged. This is designed
```

Comment:
I pushed a new version which recommends using either pulp-list or pulp-dev depending on who would be affected. There are also bullet-point guidelines recommending which to use.